protobufs/sbtc/signer/v1/backups.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+
+package sbtc.signer.v1.backups;
+
+import "bitcoin/bitcoin.proto";
+import "crypto/common.proto";
+import "crypto/wsts/wsts.proto";
+
+// Represents the verification status of DKG shares.
+// Maps to the sbtc_signer.dkg_shares_status ENUM type.
+enum DkgSharesStatus {
+    // Default value, should not be used explicitly if possible.
+    DKG_SHARES_STATUS_UNSPECIFIED = 0;
+    DKG_SHARES_STATUS_UNVERIFIED = 1;
+    DKG_SHARES_STATUS_VERIFIED = 2;
+    DKG_SHARES_STATUS_FAILED = 3;
+}
+
+// DkgShares represents the distributed key generation shares for sBTC signers
+// Maps to the sbtc_signer.dkg_shares table in PostgreSQL
+message DkgShares {
+    // Primary key: The aggregate public key produced through DKG
+    crypto.PublicKey aggregate_key = 1;
+
+    // The tweaked aggregate key used for Bitcoin script creation
+    crypto.PublicKey tweaked_aggregate_key = 2;
+
+    // Encrypted private shares of the aggregate key
+    bytes encrypted_private_shares = 3;
+
+    // Public shares used for verification
+    bytes public_shares = 4;
+
+    // The Bitcoin script_pubkey derived from the tweaked aggregate key
+    bytes script_pubkey = 5;
```

Comment:
Might be worth adding a `script_pubkey` type just for this. But I'm okay either way, the benefit doesn't seem particularly high.

Reply:
I had thought about it, but ended up in that it's just a bunch of bytes anyway...